src/styles/blocks/top-bar.scss

```diff
@@ -0,0 +1,27 @@
+.top-bar {
+  display: flex;
```

Comment:
You need to apply position sticky here according to the requirement for your top and add some padding bottom for better visibility
![98181231](https://github.com/mate-academy/layout_landing-page/assets/35075922/36b191a5-171b-415d-bfe8-359a27606720)